youtube_dl/extractor/newgrounds.py

```diff
@@ -59,47 +61,64 @@ def _real_extract(self, url):
         webpage = self._download_webpage(url, media_id)
 
         title = self._html_search_regex(
-            r'<title>([^>]+)</title>', webpage, 'title')
+            r'<title>(.+?)</title>', webpage, 'title')
+
+        try:
+            media_url = self._parse_json(self._search_regex(
+                r'"url"\s*:\s*("[^"]+"),', webpage, ''), media_id)
```

Comment:
Why not give the search a name instead of `''`, say `'media url'`? I know it was like that before, but it assists debugging since the name appears in the `-v` output.